### DISCUSSION TITLE
File/Image - добавить возможность загрузки файла по ссылке

### DISCUSSION THREAD

**Discussion by YouMixx** [moonshine-software/moonshine#368] [Ideas]
Сделать возможность загрузки файла по ссылке с стороннего ресурса.
В этом случае, файл НЕ должен сохраняться физически на сервер.
В БД должна сохраняться ссылка на сторонний ресурс и подгружаться просто с него.

Часто бывает удобно, просто всунуть картинку по ссылке, не загружая её фактически на хост.
+ лично у меня проблема, что я паршу картинки по API и сохраняю ссылку в БД на сторонний ресурс, а сейчас это не работает.

Считаю, что это рядовая функция, которой желательно находится в коробке.

**Comment by lee-to** [comment]
Больше похоже на просто текстовое поле, но отображать как картинку на индексе

**Reply by YouMixx** [reply]
Это нужно совместить с обычным File/Image, чтобы можно было загружать как и по обычному, как сейчас - т.е. картинка загружается на сервер. Так и загрузить с стороннего ресурса и это должно работать вместе.

**Reply by YouMixx** [reply]
Т.е. вот пример, как это сделано в другой библе. Очень удобно.
![image](https://github.com/moonshine-software/moonshine/assets/89249508/c5f73370-946c-4853-ac33-0be9c5d7309d)

**Reply by lee-to** [reply]
Не уверен что это нужно в коробке

**Reply by YouMixx** [reply]
Да это вроде везде есть, ну, по крайней мере много где.
На мой взгляд, это самый обычный функционал, который по умолчанию должен присутствовать.

**Reply by sashagm** [reply]
вполне разумно) в одном из своих проектов мне приходилось из старого сайта сначала выгрузить все фото ( +5к файлов) а потом в проект обратно заливать партиями 😃
